iaso/api/profiles.py

```diff
@@ -75,7 +76,12 @@ def list(self, request):
             queryset = queryset.filter(user__user_permissions__codename__icontains=perms).distinct()
 
         if location:
-            queryset = queryset.filter(user__iaso_profile__org_units__pk=location).distinct()
+            queryset = queryset.filter(
+                user__iaso_profile__org_units__pk=location,
+            ).distinct()
+
+        if org_unit_type:
+            queryset = queryset.filter(user__iaso_profile__org_units__org_unit_type__pk=org_unit_type).distinct()
```

Comment:
`user__iaso_profile__org_units__org_unit_type__pk`

What we do in the filter (for both the location and org_unit_type) is the equivalent of doing `profile.user.profile` not sure that's what we really want unless I'm missing something?